substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/ResourceAttributesView.java

```diff
@@ -0,0 +1,123 @@
+package com.oracle.svm.core.jdk;
+
+import java.io.IOException;
+import java.nio.file.ReadOnlyFileSystemException;
+import java.nio.file.attribute.BasicFileAttributeView;
+import java.nio.file.attribute.FileAttributeView;
+import java.nio.file.attribute.FileTime;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ResourceAttributesView implements BasicFileAttributeView {
+
+    private enum AttributeID
+    {
```

Comment:
Please set up the formatting according to the IDEs.md doc.

Reply:
Done.